cmd/components.go

```diff
@@ -38,7 +38,8 @@ var ComponentsCmd = &cobra.Command{
 }
 
 func init() {
-	ComponentsCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "List all Dapr components in a k8s cluster")
+	ComponentsCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Lists all Dapr components in a Kubernetes cluster")
+	ComponentsCmd.Flags().BoolP("help", "h", false, "Prints this help message")
```

Comment:
List instead of Lists

Reply:
Same comment. It is to keep consistency by using the verb in the 3rd person.